Tools/wasm/README.md

```diff
@@ -33,10 +35,12 @@ docker run --rm -ti -v $(pwd):/python-wasm/cpython quay.io/tiran/cpythonbuild:em
 
 ### Compile a build Python interpreter
 
+From within the container, run the following commands:
+
 ```shell
 mkdir -p builddir/build
 pushd builddir/build
-../../configure -C
+../../cpython/configure -C
```

Comment:
Christian: IMO, we should remove the `-C` here. You do not necessarily want to reuse the configure cache from your host inside the container.

Reply:
```suggestion
../../cpython/configure
```

Reply:
I pushed this in 330ca5ef4ac798d057b53210fdb9907f70040d76